app/routes.js

```diff
@@ -1,16 +1,17 @@
 const { paramsValidator } = require('./middlewares/schemaValidator');
-const { userSchema } = require('./schemas/user');
+const { userSchema, signInSchema } = require('./schemas/user');
 const { healthCheck } = require('./controllers/healthCheck');
 
 // album controller
 const { getAlbums, getPhotos } = require('./controllers/album');
 
 // user controller
-const { signUp } = require('./controllers/user');
+const { signUp, signIn } = require('./controllers/user');
 
 exports.init = app => {
   app.get('/health', healthCheck);
   app.get('/albums', getAlbums);
   app.get('/albums/:id/photos', getPhotos);
   app.post('/users', paramsValidator(userSchema), signUp);
+  app.post('/users/sessions', paramsValidator(signInSchema), signIn);
```

Comment:
I think this is a GET request because they're requesting information to us, what do you think?

Reply:
if we use a get, i have to send params on query and I think is weird. But tell me how you do that?